zilliqa/src/cfg.rs

```diff
@@ -52,6 +55,12 @@ pub struct NodeConfig {
     /// The location of persistence data. If not set, uses a temporary path.
     #[serde(default)]
     pub data_dir: Option<String>,
+    /// Persistence checkpoint to load.
+    #[serde(default)]
+    pub load_checkpoint: Option<Checkpoint>,
+    /// Whether to enable exporting checkpoint state snapshot files.
+    #[serde(default)]
+    pub do_snapshots: bool,
```

Comment:
Is there a reason we're using both "checkpoint" and "snapshot"? Can we use just one of the words consistently?

Reply:
Good point. In theory, in my mind, "checkpoint" - short for - "weak subjectivity checkpoint" - is the height and trusted hash at which "snapshot" files are exported. Then, when starting a node, you download a snapshot file to be able to start your node from the corresponding w.s. checkpoint's height.

 In practice, I think you're right and it just ends up inconsistent. Any preference? If not I'll stick with "checkpoint"  (so we'll e.g. be exporting checkpoints, have checkpoint files, etc.).

Reply:
Checkpoint everywhere sounds good to me :)

I agree that maybe you could call the height a "checkpoint" and the actual file a "snapshot", but I don't think we really need a separate word to describe the height.